pkg/healthchecker/types/types.go

```diff
@@ -44,7 +44,7 @@ const (
 	kubeletPortKey          = "KUBELET_PORT"
 	kubeProxyPortKey        = "KUBEPROXY_PORT"
 
-	defaultHostAddress   = "127.0.0.1"
+	defaultHostAddress   = "localhost"
```

Comment:
there is a bug in line 81 and 82

```
	kubeletHealthCheckEndpoint = fmt.Sprintf("http://%s:%s/healthz", hostAddress, kubeletPort)
	kubeProxyHealthCheckEndpoint = fmt.Sprintf("http://%s:%s/healthz", hostAddress, kubeProxyPort)

```

that url should be build using `net.JoinHostPort` to support ipv6, do you mind fixing it?

Reply:
Fixed